### ISSUE TITLE
Model training and optimization

### ISSUE THREAD

**Issue by Majdi-Aref** [Majdi-Aref/mildew-detector#3]
1. Experiment with different neural network architectures to classify the cherry leaves images.
2. Optimize the hyperparameters of the chosen neural network architecture.
3. Utilize the validation set to evaluate the performance of the trained machine learning model.
4. Fine-tune the trained machine learning model to achieve a target accuracy of 97% or higher.
